tests/gnrc_dhcpv6_client/dhcpv6_server.sh

```diff
@@ -30,6 +30,11 @@ _dhcpv6_server() {
     # only used `mktemp` with dry-run above to get temp directory name, so we
     # still need to create the directory
     mkdir -p "${TMPDIR}"
+
```

Comment:
since kea does its logging to cmd line talking about files in . -which is TMPDIR- we should provide the user/tester some information to follow
```suggestion
    echo dhcpv6_server using temporary data directoy ${TMPDIR}

```